docs/moz-addon-manager.md

```diff
@@ -0,0 +1,173 @@
+# Develop features for mozAddonManager
+
+The AMO and Discovery Pane apps use the
+[`mozAddonManager`](https://bugzilla.mozilla.org/show_bug.cgi?id=1310752)
+web API to achieve a more seamless add-on installation user experience.
+However, this API is only available to a limited list of domains.
+If you go to https://addons.mozilla.org (production) in Firefox then `mozAddonManager`
+is available on the page. If you go to a development site then it's not.
+
+## Turning on mozAddonManager for development
```

Comment:
I'm not totally following. Since the key to accessing the APIs is the host in your URL bar, installation doesn't apply, unless I misunderstood your comment? I assume you meant installing with a production [`API_HOST`](https://github.com/mozilla/addons-frontend#running-amo-for-local-development)?